web/src/utils/flows.ts

```diff
@@ -1,17 +1,21 @@
 import * as _ from 'lodash';
-import { FlowDirection, Record } from '../api/ipfix';
+import { Record } from '../api/ipfix';
 import { get5Tuple } from './ids';
 
 export const mergeFlowReporters = (flows: Record[]): Record[] => {
-  // The purpose of this function is to determine if, for a given 5 tuple, we'll look at INGRESS or EGRESS reporter
-  // The assumption is that INGRESS alone, or EGRESS alone always provide a complete visiblity however
-  // Ingress traffic will also contains pktDrop and DNS responses
-  // The logic is to index flows by 5 tuples, then for each indexed set, keep only the INGRESS if present, otherwise EGRESS
+  // The purpose of this function is to determine if, for a given 5 tuple, we'll look at INGRESS, EGRESS or INNER reporter
+  // The assumption is that INGRESS alone, EGRESS alone or INNER alone always provide a complete visiblity
+  // Favor whichever contains pktDrop and/or DNS responses
   const grouped = _.groupBy(flows, get5Tuple);
   const filtersIndex = _.mapValues(
     grouped,
     (recs: Record[]) =>
-      (recs.find(r => r.labels.FlowDirection === FlowDirection.Ingress) || recs[0]).labels.FlowDirection
+      (
+        recs.find(
+          r =>
+            r.fields.DnsId !== undefined || r.fields.PktDropBytes !== undefined || r.fields.PktDropPackets !== undefined
```

Comment:
```suggestion
            r.fields.DnsId !== undefined ||  r.fields.PktDropPackets !== undefined
```

I rely on `PktDropPackets` for this check. Same on FLP side.

Reply:
Also need to change the `Duplicated flows` tooltip if you do that change:
Currently: 
```
A flow might be reported from several interfaces, and from both source and destination nodes, making it appear several times. 
By default, duplicates are hidden in favor of Ingress traffic. Showing duplicates is not possible in Overview and Topology tabs to avoid altering metric calculations. 
Use Direction filter to switch between Ingress / Egress traffic.
```

Reply:
Also in the Direction column tooltip, we should probably clarify that it's direction related to the node

Reply:
yes, I was even thinking about changing the column name to make it more explicit.

Should we also expose `IfDirection` as filter / column ?

Reply:
+1, I created a follow-up: https://issues.redhat.com/browse/NETOBSERV-1280